src/index.html

```diff
@@ -22,6 +22,398 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        href="#"
+        class="header__logo"
+      >
+        <img
+          class="header__logo-img"
+          src="images/logo.png"
+          alt="moyo_logo"
+        />
+      </a>
+
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="nav__list-item">
+            <a
+              href="#apple"
+              class="nav__list-item-link is-active"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#samsung"
+              class="nav__list-item-link"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#smartphones"
+              class="nav__list-item-link"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#laptps"
+              class="nav__list-item-link"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#gadgets"
+              class="nav__list-item-link"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#tablets"
+              class="nav__list-item-link"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#photo"
+              class="nav__list-item-link"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="nav__list-item">
+            <a
+              href="#video"
+              class="nav__list-item-link"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+
+    <main class="main">
+      <div class="container">
+        <div
+          class="card"
+          data-qa="card"
+        >
+          <img
+            src="./images/imac.jpeg"
+            alt="imac"
+            class="card__logo"
+          />
+          <span class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </span>
+
+          <span class="card__code">Product code: 195434</span>
+          <div class="card__review">
+            <div class="card__review-stars card__review-stars--4">
+              <div class="card__review-stars-star"></div>
+              <div class="card__review-stars-star"></div>
+              <div class="card__review-stars-star"></div>
+              <div class="card__review-stars-star"></div>
+              <div class="card__review-stars-star"></div>
+            </div>
```

Comment:
You need to use stars block from the previous task
```suggestion
            <div class="stars stars--4">
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
              <div class="stars__star"></div>
            </div>
```